src/test/java/teammates/test/pageobjects/AdminEmailPage.java

```diff
@@ -67,10 +82,26 @@ public void clickTrashTab() {
         waitForPageToLoad();
     }
 
+    public String getGroupListFileKey() {
+        return this.getGroupRecipientBox().getAttribute("value");
+    }
+
+    private WebElement uploadGroupListFile() {
+        return browser.driver.findElement(By.id("adminEmailGroupReceiverList"));
+    }
+
     private WebElement getRecipientBox() {
         return browser.driver.findElement(By.id("addressReceiverEmails"));
     }
 
+    private WebElement getGroupRecipientBox() {
```

Comment:
You could use `@FindBy (id = "xxxx")` annotation instead. Sample in `AdminHomePage.java`